### ISSUE TITLE
Microsoft free version

### ISSUE THREAD

**Issue by alanrburns** [btcpayserver/btcpayserver#358]
Will there be a Microsoft free version?

**Comment by Kukks** [comment]
Feel free to go build it yourself?

**Comment by sipsorcery** [comment]
@alanrburns yes you can run btcpayserver on Windows. The only exception is the lightning software but that is not required for accepting normal Bitcoin payments.

**Comment by nguyendangdung** [comment]
> The only exception is the lightning software but that is not required for accepting normal Bitcoin payments.

@sipsorcery What is the exception?

**Comment by sipsorcery** [comment]
@nguyendangdung the lightning software (at least the implementations I've seen) only come with build instructions for linux based OS. Building on Windows is possible via emulation environments such as WSL, cygwin etc. but my point was that lightning software is currently not generally ready for the Windows OS.

**Comment by nwilson1412** [comment]
I run both Eclair and LND on Windows? Both work fine..

**Comment by sipsorcery** [comment]
@nwilson1412 natively on Windows? If so my mistake.

The instructions I've seen and builds of c-lightning and LND I've done have only been for Linux environments.

**Comment by nwilson1412** [comment]
Yep, both native, you can compile them yourself with Go (LND), or download the executables. It was definitely less documented how to get it running with Windows, but it's definitely documented for Windows Native install

**Comment by rockstardev** [comment]
Guys I'm looking to close this one - if someone can explain to me what "Microsoft free version" means (like we compile without using any tools from Microsoft, is that it?) let me know so I can keep this open.

Otherwise I'll close issue to cleanup log.
